### ISSUE TITLE
Uneven detail on classes

### ISSUE THREAD

**Issue by newobj** [ccontrols/structured-types#6]
Hi there,

Thank you for the excellent support for this library. Here's something else I noticed today:

given:

lib.ts:
```
export class Foo {
    static readonly dummy = false
}
```

test.ts:
```
import { Foo } from './lib'

export class Bar extends Foo {
}

export class Baz {
  fn() { return new Foo() }
}

export class Boz {
  fn(foo: Foo) { }
}
```

analyzing test.ts gives:

```
{
  Bar: { name: 'Bar', kind: 13, extends: [ 'Foo' ], properties: [] },
  Baz: {
    name: 'Baz',
    kind: 13,
    properties: [
      {
        name: 'fn',
        kind: 11,
        returns: {
          name: 'Foo',
          filePath: 'lib.ts',
          kind: 13,
          properties: [
            {
              static: true,
              readonly: true,
              name: 'dummy',
              kind: 3,
              value: false
            }
          ]
        }
      }
    ]
  },
  Boz: {
    name: 'Boz',
    kind: 13,
    properties: [
      {
        name: 'fn',
        kind: 11,
        parameters: [ { name: 'foo', kind: 13 } ],
        returns: { kind: 12 }
      }
    ]
  }
}
```

You can see that in the case of ```Bar``` (where we extend ```Foo```), we only get the symbol ```Foo```, nothing whatsoever about the class.

In the case of ```Baz``` (where we return ```Foo```) there's rich amount of detail about ```Foo``` (this is great and what I desire :) 

In the case of ```Boz``` (where we take ```Foo``` as a param), we get nothing, just "kind: 13"

I notice that if you change Foo to have at least one non static property, e.g. change it to

```
export class Foo {
    dummy = false
}
```

Then in the case of ```Boz``` you at least get:

```
  Boz: {
    name: 'Boz',
    kind: 13,
    properties: [
      {
        name: 'fn',
        kind: 11,
        parameters: [
          {
            name: 'foo',
            kind: 13,
            properties: [ { parent: 'Foo', name: 'dummy', kind: 3, value: false } ]
          }
        ],
        returns: { kind: 12 }
      }
    ]
  }
```

Then I sort of get an understanding of ```Foo``` that I can reconstruct, although it's not as nice as the structure given in a ```returns:``` property.

Is it possible to emit that rich set of information in all cases?

Thanks!

**Comment by atanasster** [comment]
Thanks for the detailed issue - looks like some oversight, since it works fine once you add a non-static prop.

i will try to fix it later today.

**Comment by atanasster** [comment]
Hi @newobj , I think its fixed in `3.20.4` playground: https://bit.ly/3jRB3WF

There were two issues:
1. The extends members were lacking information (I didnt think initially that more than the class name will be needed). Now the extended info is added.
2. Static props are not being returned by the ts compiler api `type.getApparentProperties` (in your example the function parameter `foo: Foo`). I fixed it for now with a workaround using a different compiler api, but also checking the ts compiler guys if this is intentional and what was the reasoning - if there was a valid reason to hide static props, I might have to revert my workaround
